### ISSUE TITLE
Recommendation_Model.Accuracy is Repeated

### ISSUE THREAD

**Issue by AugustusHsu** [AugustusHsu/Music-Recommendation-System#2]
accuracy(self) and self.accuracy is Repeated
So the accuracy is always 0 or 1
Figure out the difference between of `tf.metrics.accuracy` and the accuracy function
```
def accuracy(self):
    # Calculate accuracy
    with tf.name_scope('accuracy'):
        predictions = tf.argmax(self.Output, 1, name='predictions')
        correct_pred = tf.equal(predictions, tf.argmax(self.onehot_label, 1))
        self.accuracy = tf.reduce_mean(tf.cast(correct_pred, tf.float32))
```

**Comment by AugustusHsu** [comment]
Use `Train_sum = 0` and `Train_sum = Train_sum + train_accuracy` to sum of the train_accuracy
Print `Train_sum/(Song_X.shape[0]//setting.batch_size)` as the training accuracy of each epoch
Do the same thing at the Validation Set
